metadata.yaml

```diff
@@ -18,6 +18,7 @@ doc_title: "Concept of Control System Operations"
 # Author names, ordered as a list. Each author name should be formatted as 'First Last'
 authors:
   - "Tiago Ribeiro"
+  - "Patrick Ingraham"
```

Comment:
Do you want to add your name first? after all you did most of the work here so far.

Reply:
Sure... make me take the blame.... well played.